app.js

```diff
@@ -5,4 +5,8 @@ const { getTopics } = require("./controllers/getTopics");
 app.use(express.json());
```

Comment:
remove app.use((express.json()) from app until we need it for a post or patch request